src/features/common/LandingVideo/styles.module.scss

```diff
@@ -97,3 +97,40 @@
         transform: rotate(360deg);
     }
 }
+
+
+.playButton {
+    position: fixed;
```

Comment:
Giving this button a fixed position of course can not adapt the position to the changing width of the explore button in different languages.

Reply:
I added some space to the right site of the play button to support also longer translations.